src/ARCtrl/ARCtrl.fs

```diff
@@ -294,10 +296,12 @@ type ARC(?isa : ISA.ArcInvestigation, ?cwl : CWL.CWL, ?fs : FileSystem.FileSyste
                     yield $"{assayFoldername}/README.md"
 
                     //just allow any constructed path from cell values. there may be occasions where this includes wrong files, but its good enough for now.
-                    for (kv) in a.Tables[0].Values do
-                        yield kv.Value.AsFreeText // from arc root
-                        yield $"{assayFoldername}/dataset/{kv.Value.AsFreeText}" // from assay root > dataset
-                        yield $"{assayFoldername}/protocols/{kv.Value.AsFreeText}" // from assay root > protocols
+                    for table in a.Tables do
+                        for kv in table.Values do
+                            let textValue = kv.Value.ToFreeTextCell().AsFreeText
```

Comment:
@Freymaurer @HLWeil is this the way to access a cells text? If that's the case, I suggest we add a convenience function for this